tests/installation/reboot_after_installation.pm

```diff
@@ -103,6 +103,11 @@ sub run {
     }
     else {
         power_action('reboot', observe => 1, keepconsole => 1, first_reboot => 1);
+        # Press ESC to disable timeout of bootloader
+        # exclude remote backend where no grub_test follows reboot_after_installation
+        if (!(is_remote_backend)) {
+            send_key 'esc' if (assert_screen [qw(grub2 bootloader-grub2 bootloader)], 600);
```

Comment:
it looks good:

http://f40.suse.de/tests/7113#step/reboot_after_installation/2

it works fine with less RAM

Reply:
It didn't work as I expected. Look at the next screen: http://f40.suse.de/tests/7113#step/grub_test/1
The timeout is still there.

So, we need an `assert_screen` to be sure that the timeout is not present.

Reply:
please tell me which installation scenario doesn't include disable_grub_timeout, thanks.

Reply:
I don't remember, I would have to look for it. But, since you are working on the ticket, could you please look for it?

But, in any case, the PR is wrong as it is, because:
1. If there is no scenario at all scheduling `reboot_after_installation`, then we don't need to send the 'esc' key.
2. If we send the 'esc' key we need to ensure that the timeout is canceled.

Reply:
1. I cannot find test suite for installation without disable_grub_timeout, all above test verification contains it.
I will check this further.
2. esc key is only triggered from reboot_after_installation and if this is not 'is_remote_backend'. So we don't need to care of whether it is scheduled.

Reply:
Well, then that's it. We also need to care on remote-backend scenarios, because we have such scenarios.
We need to apply the same logic of leaving the system in an expected static state.

https://openqa.suse.de/tests/3938074#step/reconnect_mgmt_console/4
For such cases, reconnect_mgmt_console is the module for what we need to leave the system in a expected static state.

So, the first part of reconnect_mgmt_console should be done in reboot_after_installation